database/postgreSQL/schema.sql

```diff
@@ -0,0 +1,42 @@
+DROP DATABASE IF EXISTS product_options;
+
+CREATE DATABASE product_options;
+
+\c product_options;
+
+CREATE TABLE IF NOT EXISTS colors(
+  item_id INT PRIMARY KEY NOT NULL,
+  color VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS sizes(
+  item_id INT PRIMARY KEY NOT NULL,
+  size VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS items(
+  item_id INT PRIMARY KEY NOT NULL,
+  title VARCHAR(65) NOT NULL,
+  original_price DECIMAL NOT NULL,
+  sale_price DECIMAL NOT NULL,
+  _description VARCHAR(255) NOT NULL,
+  liked BOOLEAN NOT NULL,
+  in_stock INT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS reviews(
+  item_id INT NOT NULL,
```

Comment:
you can use the item_id to link them as a foreign key? unless youre plannign on making multiple queries which might be easier to implement